### ISSUE TITLE
Feature Request - Launcher

### ISSUE THREAD

**Issue by FreshImmuc** [AliaksandrSiarohin/first-order-model#387]
Hello, please code a Program (exe) or a GUI for the whole process.

**Comment by instant-high** [comment]
I've coded a GUI for it (VB6) and added some extra features by modifying the python script.
Search on youtube "GUI for First Order Motion Animation"
not sure if it would work for everyone. 
I use a ini-file which has to be edited manually, some paths...

**Comment by FreshImmuc** [comment]
ok but where can i download it? can u send a link?

**Comment by instant-high** [comment]
Ok.
Added download link in video description. 
Made some changes to the program, read the included "!FOM_GUI.txt" how to install
Good luck if you try (let me know if it works for you)

**Comment by FreshImmuc** [comment]
Hey, i installed anaconda, Extracted the ZIP into the GitHub Folder, overwritten the files, that already existed, added all Paths. but then i got an error: 

![image](https://user-images.githubusercontent.com/65733898/110252367-54803b80-7f85-11eb-9cea-f5a7c92035cf.png)


I tried installing PyTorch, cause thats the only thing i found by searching: "torch download" ... How do i get "Torch" ?

**Comment by instant-high** [comment]
I don't think that this has something to do with my "!Animation.py". I only added some other functions.
Have you tried the original "demo.py" ? It should produce the the same error....

**Comment by FreshImmuc** [comment]
What do you mean with the 1st line? I dont understand... U mean that the error is not caused by the !Animation.py?

**Comment by FreshImmuc** [comment]
Ill try to run the demo.py

**Comment by FreshImmuc** [comment]
I tried to run it but i got this error:

![image](https://user-images.githubusercontent.com/65733898/110255070-4553ba80-7f92-11eb-941f-b8f9e92dd6f5.png)

**Comment by FreshImmuc** [comment]
How do I install these modules? Maybe i just installed them wrong

**Comment by instant-high** [comment]
I've added some functions to demo.py and renamed it !animation.py
The FOM_GUI writes a batch file that executes !animation.py instead of demo.py
Import Torch is the same in both of them

**Comment by FreshImmuc** [comment]
Okay, and that means what? How can I fix it? Sorry but i dont understand that much of programming haha

**Comment by instant-high** [comment]
Animation Demo
To run a demo, download checkpoint and run the following command:

python demo.py  --config config/dataset_name.yaml --driving_video path/to/driving --source_image path/to/source --checkpoint path/to/checkpoint --relative --adapt_scale

**Comment by FreshImmuc** [comment]
How can i download the checkpoint

**Comment by FreshImmuc** [comment]
I ran it and i got same error

**Comment by FreshImmuc** [comment]
![image](https://user-images.githubusercontent.com/65733898/110255290-37526980-7f93-11eb-85f7-3898041aa3a9.png)

**Comment by instant-high** [comment]
Look at the code page. There are two download-links for pretrained models.
Put them in a folder named checkpoints
Put the *.yaml in folder config.
Try demo.py and FOM_Gui again.

**Comment by instant-high** [comment]
Did you install first order model into an environment? What is the name of your environment.
Look at the first picture. Could not find anacondaenv . Did you change fom_gui.ini correctly? ...activate "environmentname"

**Comment by FreshImmuc** [comment]
Okay, sorry for "late" response, hope you are still online haha, 

This is what I did in the Installation process:

1. Downloaded all files
2. Downloaded GitHub Folder and put it into the other folder
3. extracted the python_new.zip file into the GitHub Folder and replaced existing files
4. installed Anaconda3
5. Configured the fom_gui.ini file

> **fom_gui.ini File:**
> call D:\Programms\Anaconda3\Scripts\activate.bat
> call conda activate Anacondaenv
> cd D:\Desktop\Tools\Deepfakes\FOM_GUI_1\first-order-model-master
> D:\Desktop\Tools\Deepfakes\FOM_GUI_1\first-order-model-master

6. I think 6. is the Problem, cause I did it wrong. I just used Miniconda as promt, not Anaconda promt... I installed Miniconda a while ago and I used that now, cause its the only thing that shows up if I search "Anaconda". How do I open the Normal Anaconda Promt instead?
![image](https://user-images.githubusercontent.com/65733898/110259158-979ed680-7fa6-11eb-96d5-dcbc48ea0ce4.png)
I did some other things after this but I think 6. is done wrong. Also the envs folder in the Anaconda3 Folder (wich i think is the folder where all Enviroments are located) is empty..
![image](https://user-images.githubusercontent.com/65733898/110259245-f06e6f00-7fa6-11eb-909b-02de10fb2d2a.png)

**Comment by instant-high** [comment]
Don't know if it works if not installed in an environment. 
Did you "pip install -r requirements.txt" ?
I followed these instructions : https://www.youtube.com/watch?v=ICvNnrWKHmc
Some was quite different but I got it to work. 
Changed my Github mail address to public. Maybe you can write a mail and I send you more instructions
